From 027ca2530ce94dd7d9954e57631aa34987db392e Mon Sep 17 00:00:00 2001
From: Ying Zhang <ying.zhang22455@nxp.com>
Date: Thu, 29 Sep 2016 11:38:36 +0800
Subject: [PATCH 120/124] armv8: aarch32: defconfig: Enable firmware loading

As some modules require userspace firmware loading support, such as
PPFE, add this feature in the defconfig for AArch32 on ARMv8.

Signed-off-by: Alison Wang <alison.wang@nxp.com>
---
 arch/arm/configs/ls_aarch32_defconfig |    1 -
 1 file changed, 1 deletion(-)

--- a/arch/arm/configs/ls_aarch32_defconfig
+++ b/arch/arm/configs/ls_aarch32_defconfig
@@ -71,7 +71,6 @@ CONFIG_CAN=y
 CONFIG_CAN_FLEXCAN=y
 CONFIG_DEVTMPFS=y
 CONFIG_DEVTMPFS_MOUNT=y
-# CONFIG_FW_LOADER is not set
 CONFIG_DMA_CMA=y
 CONFIG_MTD=y
 CONFIG_MTD_CMDLINE_PARTS=y
